static/app/views/alerts/rules/metric/ruleConditionsForm.tsx

```diff
@@ -653,4 +783,46 @@ const FormRow = styled('div')<{columns?: number; noMargin?: boolean}>`
     `}
 `;
 
+const MonitorSelect = styled('div')`
+  border-radius: ${p => p.theme.borderRadius};
+  border: 1px solid ${p => p.theme.border};
+  width: 100%;
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+`;
+
+type MonitorCardProps = {
+  isSelected: boolean;
+  /**
+   * Adds hover and focus states to the card
+   */
+  position: 'left' | 'right';
+};
+
+const MonitorCard = styled('div')<MonitorCardProps>`
+  padding: ${space(1)};
+  display: flex;
+  flex-grow: 1;
+  flex-direction: column;
+  cursor: pointer;
+
+  &:focus,
+  &:hover {
+    outline: 1px solid ${p => p.theme.purple200};
+    background-color: ${p => p.theme.backgroundSecondary};
+  }
+
+  border-top-left-radius: ${p => (p.position === 'left' ? p.theme.borderRadius : 0)};
+  border-bottom-left-radius: ${p => (p.position === 'left' ? p.theme.borderRadius : 0)};
+  border-top-right-radius: ${p => (p.position !== 'left' ? p.theme.borderRadius : 0)};
+  border-bottom-right-radius: ${p => (p.position !== 'left' ? p.theme.borderRadius : 0)};
```

Comment:
Since you're only using 1 radius size, i'd just use `border-radius` instead.
```suggestion
  border-radius: ${p => (p.position === 'left' ? p.theme.borderRadius : 0)};
```

Reply:
Oh hmmm
but i'm modifying different radiuses - not _all_ of them

this is what lets me highlight the radius'd corners, but have 90's between the cards
![Screenshot 2024-04-16 at 2 07 58 PM](https://github.com/getsentry/sentry/assets/6186377/bc5dbd6f-94ea-4add-8dfd-b85f13c1fc83)

Reply:
Ah - nvm.